Types.ts

```diff
@@ -471,20 +471,28 @@ export enum LinkEventName {
 }
 
 export enum LinkEventViewName {
+    ACCEPT_TOS = 'ACCEPT_TOS',
     CONNECTED = 'CONNECTED',
     CONSENT = 'CONSENT',
     CREDENTIAL = 'CREDENTIAL',
+    DOCUMENTARY_VERIFICATION = 'DOCUMENTARY_VERIFICATION',
     ERROR = 'ERROR',
     EXIT = 'EXIT',
+    KYC_CHECK = 'KYC_CHECK',
+    LIVENESS = 'LIVENESS',
```

Comment:
I am unclear on what this PR is doing generally, but "liveness" is a deprecated name--anything that might be consumable should be 'SELFIE_CHECK'. Are we still emitting the wrong name somewhere?